test/upload_test.go

```diff
@@ -212,6 +212,28 @@ func TestRateLimit(t *testing.T) {
 
 }
 
+func TestContainsSyntheticMedia(t *testing.T) {
+	url, err := url.Parse(testServer.URL)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	transport := &mockTransport{url: url}
+	limitTransport, err := limiter.NewLimitTransport(transport, limiter.LimitRange{}, fileSize, 0)
+	if err != nil {
+		t.Fatal(err)
+	}
+	config.ContainsSyntheticMedia = true
+	ctx := context.Background()
+	videoReader := &mockReader{fileSize: fileSize}
+	defer videoReader.Close()
+
+	err = yt.Run(ctx, limitTransport, config, videoReader)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
```

Comment:
I appreciate you making the effort to create a test, however this test doesn't really achieve anything useful. Please remove it. (I don't think it's really necessary to have tests for all the video properties)